features_module_how_to_share_sites_config_with_oth.md

```diff
@@ -0,0 +1,118 @@
+# Features module: how to share site’s config with others?
+
+If you already had a chance to build one or two Drupal sites,  you probably know Drupal site consists of three parts:
```

Comment:
I really liked the picture you drew on the board.  There is a lot of information given in text so if there is any way to move text into visuals, I am all for it.